### DISCUSSION TITLE
Quando clico no profile ele nao carrega para a pagina profile-page

### DISCUSSION THREAD

**Discussion by marquesleo** [balta-io/7187#56] [Perguntas e Respostas] [answered]
quando clico no icone do profile nao vai para a tela de profile-page,
ja vi as rotas no app-routing.module.ts
vc tem alguma luz ? 
qdo coloco na barra de endereco url + /account 
aparece a pagina mas sem o botao salvar visivel

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite @marquesleo, posta alguns códigos ai pra ajudar... assim sem ver nada não consigo ajudar!
